src/tritonserver.cc

```diff
@@ -1215,6 +1215,20 @@ TRITONSERVER_ServerOptionsSetModelLoadThreadCount(
   return nullptr;  // Success
 }
 
+TRITONAPI_DECLSPEC TRITONSERVER_Error*
+TRITONSERVER_ServerOptionsSetLogOutFile(
+    TRITONSERVER_ServerOptions* options, const char* file)
+{
+#ifdef TRITON_ENABLE_LOGGING
+  std::string out_file(file);
```

Comment:
Do we need to add a check here to make sure that the log file is not empty and return an error if it is empty?

Reply:
I believe that should be handled here: https://github.com/triton-inference-server/common/pull/66/commits/efd0df9d0a2fac31918673b462cc3635f386ba37#diff-6215e044cfaca75f9bf9a1e1156e1cb21ce7085706b68b5d1b771ace1fc8f8fcR57-R67. Is this sufficient?

Reply:
I was wondering does setting the file to an empty string mean that we will be redirecting the logs to output? Do we need to document this behavior?

Reply:
No, empty string is the default. If the user enters the empty string, triton will redirect logging facilities to the console. It is documented here: https://github.com/triton-inference-server/server/pull/4640/files#diff-b2f94db5cc9d55373c593406216c0c9815a3bf8d10275fa60d76006346c988a3R71-R72

However, I can specifically mention the behavior of the empty string if you think it would be best?

Reply:
https://github.com/triton-inference-server/core/pull/98/files#r934647392